programmers/3일차/자릿수-더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(N) {
+    const stringofN = String(N);
```

Comment:
```suggestion
    const str = String(N);
```
이게 더 좋을거 같아요  N을 문자열로 변환했다고 표시하신것 같은데 N은 그냥 입력값이고 언제든지 변할 수 있는 값이기 때문에  변수명에는 굳이 반영하지 않으셔도 될거같아요